### ISSUE TITLE
[React][Menu] Invalid aria-controls attribute with lazyMount

### ISSUE THREAD

**Issue by fseglard** [chakra-ui/ark#1508]
### Description

Hi!

When using the `lazyMount` attribute on the `MenuContent` element, it causes the `aria-controls` attribute to be invalid on the `MenuTrigger` element when using `jest-axe`:

```
"ARIA attributes must conform to valid values (aria-valid-attr-value)"

Fix all of the following:
  Invalid ARIA attribute value: aria-controls="menu::r0::content"
```

If I remove the `lazyMount` property, everthing is ok.

I don't really understand why it's happening because when I use the same check on the `Select` component, with lazy mounting enabled, I don't have any errors.

### Link to Reproduction

https://codesandbox.io/s/damp-paper-sym44g?file=/src/App.js

### Steps to reproduce

1. Go to https://codesandbox.io/s/damp-paper-sym44g?file=/src/App.js
2. Run the test
3. Check the error

Then remove the `lazyMount` attribute on the `MenuContent`, and re-run the test: all green.

### Ark UI Version

0.15.0

### Framework

- [X] React
- [ ] Solid
- [ ] Vue

### Browser

Google Chrome 117

**Comment by cschroeter** [comment]
@fseglard 

Thanks for taking your time to report this issue. A fix will be availabe in the next release https://github.com/chakra-ui/ark/commit/fb7441d44127df0638bada4cdfebaacd489fbee1

**Comment by fseglard** [comment]
@cschroeter 

After testing the version 1.0.0 of ArkUI, I still have the issue with the `Menu` component and accessibility.

Here's the reproduction link: https://codesandbox.io/s/festive-smoke-zn3w2n?file=/src/Menu.test.tsx.

As you can see, there is still a problem with the `aria-controls` attribute on the `Menu` component. Also, I've created another test example with the  `Select` component with the exact same integration and there is no problem with this one.

**Comment by cschroeter** [comment]
@fseglard 

This should be addressed with https://github.com/chakra-ui/ark/commit/42667493eb1c398ddca297ffbe42698cc2a9930a

A fix will be avilable in 1.0.1
